bin/pygrb/pycbc_pygrb_plot_chisq_veto

```diff
@@ -188,12 +187,10 @@ parser.add_argument("-z", "--zoom-in", default=False, action="store_true",
 parser.add_argument("-y", "--y-variable", required=True,
                     choices=['standard', 'bank', 'auto'],
                     help="Quantity to plot on the vertical axis.")
-parser.add_argument("--use-sngl-ifo-snr", default=False,
-                    action="store_true", help="Plots are vs single IFO " +
-                    "SNR, rather than coherent SNR")
-parser.add_argument("--use-sngl-ifo-veto", default=False,
-                    action="store_true", help="Single IFO veto values " +
-                    "plotted, rather than network ones")
```

Comment:
Why get rid of this?  We do want to keep the ability of looking at single IFO chi-square values.